### PR TITLE
[SMALLFIX] Clean up in core module

### PR DESCRIPTION
**Pull request by HorizonNet**
Small clean up of some classes in the *core* module.

### CODE REVIEW

File: core/server/src/main/java/alluxio/worker/block/evictor/AbstractEvictor.java [hunk 1]
```diff
@@ -125,7 +125,7 @@ protected StorageDirView cascadingEvict(long bytesToBeAvailable, BlockStoreLocat
           BlockMeta block = mManagerView.getBlockMeta(blockId);
           if (block != null) {
             candidateDirView.markBlockMoveOut(blockId, block.getBlockSize());
-            plan.toEvict().add(new Pair<Long, BlockStoreLocation>(blockId,
+            plan.toEvict().add(new Pair<>(blockId,
                 candidateDirView.toBlockStoreLocation()));
```
Comment (uronce-cc) [inline core/server/src/main/java/alluxio/worker/block/evictor/AbstractEvictor.java]: NIT: Can fit in the above line now?

File: core/server/src/main/java/alluxio/worker/block/evictor/AbstractEvictor.java [hunk 2]
```diff
@@ -149,7 +149,7 @@ protected StorageDirView cascadingEvict(long bytesToBeAvailable, BlockStoreLocat
           if (nextDirView == null) {
             // If we failed to find a dir in the next tier to move this block, evict it and
             // continue. Normally this should not happen.
-            plan.toEvict().add(new Pair<Long, BlockStoreLocation>(blockId,
+            plan.toEvict().add(new Pair<>(blockId,
                 block.getBlockLocation()));
```
Comment (uronce-cc) [inline core/server/src/main/java/alluxio/worker/block/evictor/AbstractEvictor.java]: NIT: Can fit in the above line now?

File: core/server/src/main/java/alluxio/worker/block/evictor/GreedyEvictor.java [hunk 3]
```diff
@@ -98,15 +98,15 @@ public EvictionPlan freeSpaceWithView(long availableBytes, BlockStoreLocation lo
...
     for (BlockMeta block : victimBlocks) {
       // TODO(qifan): Should avoid calling getParentDir.
       String fromTierAlias = block.getParentDir().getParentTier().getTierAlias();
       List<StorageTierView> candidateTiers = view.getTierViewsBelow(fromTierAlias);
       StorageDirView dstDir = selectAvailableDir(block, candidateTiers, pendingBytesInDir);
       if (dstDir == null) {
         // Not possible to transfer
-        toEvict.add(new Pair<Long, BlockStoreLocation>(block.getBlockId(),
+        toEvict.add(new Pair<>(block.getBlockId(),
             block.getBlockLocation()));
```
Comment (uronce-cc) [inline core/server/src/main/java/alluxio/worker/block/evictor/GreedyEvictor.java]: NIT: Can fit in the above line now?

**Comment by uronce-cc** [comment]
@HorizonNet Thanks for the cleanup! I left some minor comments.

**Comment by HorizonNet** [comment]
@uronce-cc Thanks for the comments. I updated the PR accordingly.

**Comment by uronce-cc** [comment]
LGTM

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**